lib/fs/fakefs_test.go

```diff
@@ -796,6 +800,9 @@ func testFakeFSSameFile(t *testing.T, fs Filesystem) {
 			t.Fatalf("Could not create %s: %s", filename, err)
 		} else {
 			fd.Close()
+			if runtime.GOOS == "windows" {
+				time.Sleep(1 * time.Millisecond)
+			}
```

Comment:
We should not need this, I'd like to see this fails and repro it.

Reply:
This does fail without this sleep on our build server. I'll remove it to demostrate.